### PR TITLE
[Docs] Minor MeshModel Changes

### PR DESCRIPTION
**Pull request by Boombag0607**
**Notes for Reviewers**

See PR #8182 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d157b8ff28ce605e19811e64645a034e8a8ca276 at: https://64b221c241c2c54bd9fabcc8--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-meshmodel.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ In the context of MeshModel, the `Core Constructs` of MeshModel are represented
 
 If a specific attribute is not provided with a value in the definition, it means that the value for that attribute has to be written or configured per construct. In other words, the absence of a value indicates that the configuration for that attribute is required and specific to each individual construct instance.
 
-[![Meshery architecture]({{ site.baseurl }}/assets/img/meshmodel/meshmodel-architecture.svg)]({{ site.baseurl }}/assets/img/meshmodel/meshmodel-architecture.svg)
+[![MeshModel Contruct Classification_](https://raw.githubusercontent.com/meshery/meshery/master/docs/assets/img/meshmodel/meshmodel-architecture.svg)](https://raw.githubusercontent.com/meshery/meshery/master/docs/assets/img/meshmodel/meshmodel-architecture.svg)
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-meshmodel.md:24]: It's ok, in fact ideal, if you use `{{ site.baseurl }}`.

**Comment by l5io** [comment]
🚀 Preview for commit 1a9fa6a5ae210e2bd4ef9a61a64f8232523aefaf at: https://64b389198ca8dd046832a8cf--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1a9fa6a5ae210e2bd4ef9a61a64f8232523aefaf at: https://64b389192458f87f2743a1d2--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e9f5f54d504b1d6d816a916d52a4aa82019b3503 at: https://64b38a91dd121d015d4e7ab5--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit efe443b2489085e8dc4e3c929c7034fde11e43ce at: https://64b41f44bd75e955b6defafd--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5161f62b3d7120d81d3e14df98f844c11a90acc1 at: https://64b4e7c8b4979e3f6e766047--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5161f62b3d7120d81d3e14df98f844c11a90acc1 at: https://64b4e7e3b4ec053a7fd6c13d--meshery-docs-preview.netlify.app

**Pull request merged**